.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+### Requirements
+* Read the [HyperSpy developer guide](https://hyperspy.org/hyperspy-doc/current/dev_guide/intro.html).
```

Comment:
Once there is a contributor guide in the `RosettaSciIO` docs (possibly with a link to the hyperspy developer guide or other), we should update this link, but in the meantime, this is useful as a reference!

Reply:
Yes, that was the idea :-)